inc/change.class.php

```diff
@@ -636,9 +636,17 @@ function showForm($ID, $options=array()) {
          $showuserlink = 1;
       }
 
+      if (!$this->isNewItem()) {
+         $options['formtitle'] = sprintf(
+            __('%1$s - ID %2$d'),
+            $this->getTypeName(1),
```

Comment:
It's not a correct gettext sentence.
You must use a sentence for ID %2$d for language readied from right to left

Reply:
So this is alredy warong in some other places (commondbtm, commonglpi, ...)